lib/models/blueprint.js

```diff
@@ -59,14 +59,14 @@ const writeFile = RSVP.denodeify(fs.outputFile);
   `files` contains templates for the all the files to be
   installed into the target directory.
 
-  The `__name__` token is subtituted with the dasherized
+  The __`__name__`__ token is subtituted with the dasherized
```

Comment:
well, I understand what it does, but what is not clear to me is why we would want it explicitly bold here. we could do the same using CSS if we wanted to.